qa/automation/CareerPortal.Tests.Scenarios/Savostin/buttons_of_Career_Quantori_page.feature

```diff
@@ -0,0 +1,48 @@
+Feature: 1.0.0 All buttons of "Career Quantori" page lead to correct pages
```

Comment:
The name of the feature file should be the equal name of the feature inside of file.
Feature: 1.0.1 Header/Footer of Career Portal

Also, a bad practice to try to test everything in one feature file.
The functional of the Career Portal can be separated into several sections : 
- Header 
- Footer
- Filter of positions 
- List of positions
- Learn more section
- Hiring Process
- CV form

And why do you use the same number of features? 
You already used the number 1.0.0